records/2022-03-08.priority-lanes.md

```diff
@@ -16,7 +16,7 @@ In order to buffer up notifications between the API and the celery tasks, GCNoti
 
 ## Current notification pipeline flow
 
-![Current notification flow](https://raw.githubusercontent.com/cds-snc/notification-adr/main/records/diagrams/2022-03-08.priority-lanes/current-lanes-setup.drawio.png?raw=true "Current notification flow")
+![Current notification flow](diagrams/2022-03-08.priority-lanes/current-lanes-setup.drawio.png?raw=true "Current notification flow")
```

Comment:
Let's try the relative link..

Reply:
Seems to work... well in the PR anyway! here goes... 🚀